docs/operator-manual/notifications/examples.md

```diff
@@ -0,0 +1,91 @@
+Here you can find some examples of what you can do with the notifications service in ArgoCD.
+
+## Getting notified when a sync occurs and understanding how your resources changed
+
+With ArgoCD you can build a notification system that tells you when a sync occurred and what it changed. 
+To get notified via webhook when a sync occurs you can add the following trigger:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-notifications-cm
+data:
+  service.webhook.on-deployed-webhook: |
+    url: <your-webhook-url>
+    headers:
+    - name: "Content-Type"
+      value: "application/json"
+
+  template.on-deployed-template: |
+    webhook:
+      on-deployed-webhook:
+        method: POST
+        body: |
+              {{toJson .app.status.operationState.syncResult}}
+
+  trigger.on-deployed-trigger: |
+    when: app.status.operationState.phase in ['Succeeded'] and app.status.health.status in ['Healthy', 'Degraded']
+    oncePer: app.status.sync.revision
+    send: [on-deployed-template]
+```
+
+This, as explained in the [triggers section](triggers/#avoid-sending-same-notification-too-often), will generate a notification when the app is synced and healthy. We then need to create a subscription for the webhook integration:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  annotations:
+    notifications.argoproj.io/subscribe.on-deployed-trigger.on-deployed-webhook: ""
+```
+
+You can test that this works and see how the response looks by adding any webhook site and syncing our application. Here you can see that we receive a list of resources, with a message and some properties of them. For example:
+
+```json
+{
+  "resources": [
+    {
+      "group": "apps",
+      "hookPhase": "Running",
+      "kind": "Deployment",
+      "message": "deployment.apps/test configured",
+      "name": "test",
+      "namespace": "argocd",
+      "status": "Synced",
+      "syncPhase": "Sync",
+      "version": "v1"
+      images: [
+        "nginx:1.27.1"
+      ]
```

Comment:
Add quotation marks to key and move into alphabetical order.

Would this be a good place to document whether the `images` array is sorted?